odbc/src/api/data_tests.rs

```diff
@@ -789,12 +784,33 @@ mod unit {
                 );
                 bin_val_test(I32_COL, &[1, 0, 0, 0], SqlReturn::SUCCESS);
                 bin_val_test(I64_COL, &[0, 0, 0, 0, 0, 0, 0, 0], SqlReturn::SUCCESS);
-                bin_val_test(JS_COL, &[], SqlReturn::ERROR);
-                bin_val_test(JS_W_S_COL, &[], SqlReturn::ERROR);
-                bin_val_test(MAXKEY_COL, &[], SqlReturn::ERROR);
-                bin_val_test(MINKEY_COL, &[], SqlReturn::ERROR);
-                bin_val_test(OID_COL, &[], SqlReturn::ERROR);
-                bin_val_test(REGEX_COL, &[], SqlReturn::ERROR);
+                bin_val_test(
+                    JS_COL,
+                    "{\"$code\":\"log(\\\"hello world\\\")\"}".as_bytes(),
+                    SqlReturn::SUCCESS,
+                );
+                bin_val_test(JS_W_S_COL, "{\"$code\":\"log(\\\"hello\\\" + x + \\\"world\\\")\",\"$scope\":{\"x\":{\"$numberInt\":\"42\"}}}".as_bytes(), SqlReturn::SUCCESS);
+                bin_val_test(
+                    MAXKEY_COL,
+                    &[123, 34, 36, 109, 97, 120, 75, 101, 121, 34, 58, 49, 125],
```

Comment:
Nice work on the test cases.  For MaxKey and MinKey we should also be able to represent them as:
"{\"$maxKey\":1}".as_bytes()
"{\"$minKey\":1}".as_bytes()